elements/bernardi-guzman-neilan-raugel.def

```diff
@@ -0,0 +1,50 @@
+name: Bernardi-Guzman-Neilan-Raugel
+html-name: Bernardi&ndash;Guzm&aacute;n&ndash;Neilan&ndash;Raugel
+min-degree: 1
+max-degree: 1
+short-names:
+  - BGNR
+ndofs:
+  triangle:
+    formula: 11
+  tetrahedron:
+    formula:
+    - k=1: 19
+    - k=2: 49
+categories:
+  - vector
+  - Hdiv
+  - macro
+reference-elements:
+  - triangle
+  - tetrahedron
+dofs:
+  edges: (if \(k>1\)) point evaluations in tangential directions at midpoints
+  facets:
+    - point evaluations in normal directions at vertices
+    - normal integral moments with (lagrange,0)
+    - (if \(k>1\)) point evaluations in normal directions at midpoints of edges
+  cell:
+    - point evaluations at the midpoint
+mapping: contravariant Piola
+sobolev: H(div)
+notes:
+  - This element is a modification of the [Bernardi&ndash;Raugel](element::bernardi-raugel) element with the facet bubbles modified to be divergence free.
```

Comment:
This element is BR with modified bubbles, plus interior (macro) bubbles: the hat function associated to the barycenter for each coordinate direction.